spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/web/OwnerResource.java

```diff
@@ -40,13 +41,15 @@
 class OwnerResource {
 
     private final OwnerRepository ownerRepository;
+    private final MeterRegistry registry;
 
     /**
      * Create Owner
      */
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
     public void createOwner(@Valid @RequestBody Owner owner) {
+        registry.counter("create.owner").increment();
```

Comment:
What about using [@Timed](https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/annotation/Timed.java#L24)? If we use it instead of manually incrementing metrics, tests will be alse simplified. IMHO there is little profit in measuring number of requests - Prometheus has metrics for RPS etc.)

https://spring.io/blog/2018/03/16/micrometer-spring-boot-2-s-new-application-metrics-collector

Reply:
Agreed. I'm delivering the workshop today so once that's done I'll make this change.

Reply:
@notsureifkevin do you any time to make the change?

Reply:
https://micrometer.io/docs/concepts#_the_code_timed_code_annotation

It's not clear to me how to use this annotation and I'm constrained for time as-is. You're welcome to use merge this PR as-is, attempt to make these modifications yourself, or drop it completely. I've rebased atop master, so it should be good to go as-is.

Reply:
@mszarlinski I propose to merge this PR and create an issue for improvment. I could work on it.

Reply:
I've created the https://github.com/spring-petclinic/spring-petclinic-microservices/issues/120 in order to merge this PR then improve it.